js/api/PopulateLocalParseServer.js

```diff
@@ -0,0 +1,140 @@
+// This file will populate your local Parse Server.
+// It fetches copies of the remote test server whenever yours is empty.
+
+import Parse from 'parse/react-native'
+import Store from '../data/Store'
+import DummyData from '../data/DummyData'
+
+import { loadSurveyList, storeSurveys } from '../api/Surveys'
+import { loadForms, storeForms } from '../api/Forms'
+import { loadQuestions, storeQuestions } from '../api/Questions'
+import { loadTriggers, storeTriggers } from '../api/Triggers'
+
+// Attempts to create interactive dummy data in a local server.
+export function initializeLocalParseData () {
+  if ( Store.server === 'remote-test' ) {
+    loadSurveyList({})
+  } else {
+    loadSurveyList({}, createInitialParseData)
+  }
+}
+
+function createInitialParseData (error, results) {
+  if (error) {
+    console.warn(error)
+  } else if (results.length === 0) {
+    console.log('Creating initial Parse server data...')
+    for (i = 0; i < DummyData.surveys.length; i++) {
+      createSurvey(DummyData.surveys[i])
+    }
+  }
+}
+
+function createSurvey(surveyData) {
+  const Survey = Parse.Object.extend('Survey')
+  let newSurvey = new Survey()
+
+  newSurvey.set('title', surveyData.title)
+  newSurvey.set('user', surveyData.user)
+  newSurvey.set('createdAt', surveyData.created)
+
+  newSurvey.save(null, {
+    success: function(response) {
+      console.log('New Survey created with objectId: ' + response.id)
+      createForms(response)
+      storeSurveys(response)
+    },
+    error: function(response, error) {
+      console.warn('Failed to create Survey, with error code: ' + error.message)
+    }
+  })
+}
+
+function createForms(parentSurvey) {
+  const Form = Parse.Object.extend('Form')
+  let newForm = new Form()
+
+  newForm.set('survey', parentSurvey)
```

Comment:
We are actually storing the relation on the survey class—surveys have many forms.
This is how we are doing it in the admin area: https://github.com/ecohealthalliance/mobile-survey/blob/pseudo-master/app/server/methods.coffee#L60-L62